jwst/resample/gwcs_drizzle.py

```diff
@@ -381,9 +380,13 @@ def dodrizzle(insci, input_wcs, inwht, output_wcs, outsci, outwht, outcon,
 
     # Compute the mapping between the input and output pixel coordinates
     # for use in drizzle.cdrizzle.tdriz
+
+    # pixmap = utils.calc_pixmap(input_wcs, output_wcs)
     pixmap = resample_utils.calc_gwcs_pixmap(input_wcs, output_wcs, insci.shape)
+
     # inwht[np.isnan(pixmap[:,:,0])] = 0.
```

Comment:
you didn't modify this, but is there any reason this commented line needs to stay in this file>